distributed/distributed.yaml

```diff
@@ -170,6 +170,8 @@ distributed:
     log-length: 10000  # default length of logs to keep in memory
     log-format: '%(name)s - %(levelname)s - %(message)s'
     pdb-on-err: False       # enter debug mode on scheduling error
+pickle:
+  protocol: null  # specify the pickle protocol to use
```

Comment:
Just placed this at the global level as it is not really worker or scheduler specific. Not picky on where it lives though.